aux/vagrant-build/Vagrantfile

```
@@ -6,22 +6,32 @@ SHELLARGS << (ENV['branch'] || '')
 SHELLARGS << (ENV['proxy_repo'] || '')
 
 Vagrant.configure("2") do |config|
-  config.vm.define "el7", primary: true do |machine|
+  config.vm.define "fdi-builder", primary: true do |machine|
     machine.vm.hostname = "fdi-builder-vm"
     machine.vm.provision :shell, :path => 'build_image.sh', :args => SHELLARGS
 
-    config.vm.provider :libvirt do |lv, cfg|
-      cfg.vm.box = 'centos/7'
-      lv.memory = 4096
-      lv.cpus = 2
+    config.vm.provider :libvirt do |domain, cfg|
+      cfg.vm.box = 'centos/stream8'
+      domain.memory = 7990
+      domain.cpus = 2
+      domain.nested = true
+      domain.disk_driver :cache => 'unsafe'
     end
+  end
+end
+
+# TODO: DELETE ME after https://github.com/theforeman/jenkins-jobs/pull/162/files is merged
+Vagrant.configure("2") do |config|
```

For reference, you can do it in a single definition:
```suggestion

  # TODO: DELETE ME after https://github.com/theforeman/jenkins-jobs/pull/162/files is merged
```

---

Ok dropping I think I am really close, I tested this locally and in Vagrant too, builds fine.